app/helpers/alert_notifier_helper.rb

```diff
@@ -0,0 +1,66 @@
+module AlertNotifierHelper
+  include ActionMailerThemer
+
+  def host_and_protocol_for_theme(theme)
+    { host: host(theme), protocol: protocol(theme) }
+  end
+
+  def base_tracking_params
+    { utm_source: "alerts", utm_medium: "email" }
+  end
+
+  def application_url_with_tracking(theme: nil, id: nil)
+    base_params = host_and_protocol_for_theme(theme)
+                       .merge base_tracking_params
```

Comment:
Yep I agree, it does seem a bit weird. I'll add the parentheses and move onto one line as:

    base_params = host_and_protocol_for_theme(theme).merge(base_tracking_params)